doc/4-instrument/README.md

```diff
@@ -139,6 +139,11 @@ in all cases except bitmask macros, right-clicking on the graph opens up an edit
   - **Y**: multiplies all values by the scale factor, clipping them if they would exceed the allowed range.
 - **randomize**: replaces all values with random values between **Min** and **Max**.
 
+while the pointer is over a macro view, the scroll wheel can change the view:

```

Comment:
Pointer, cursor, mouse, erm. Consistency.  Use "Mouse cursor".
Macro view... Use "Sequence view" maybe?

Awkward phrasing demands its own removal: "the scroll wheel can change the view".

Reply:
Wording changed.